src/Illuminate/Support/Collection.php

```diff
@@ -1266,6 +1266,25 @@ public function toBase()
         return new self($this);
     }
 
+    public function __get($name)
+    {
+        $nameToMethod = [
+            'where' => 'filter',
+            'unless' => 'reject',
+            'do' => 'each',
+            'extract' => 'map',
+            'sum' => 'sum',
+            'inOrderOf' => 'sortBy',
+            'inReverseOrderOf' => 'sortByDesc',
+        ];
```

Comment:
That was just the original implementation with the method names from the article that I linked in the original description. I didn't consider this finished, but Taylor merged it and changed it to what was explained in the Laravel News article.